portal-api/api-tests/v1/gef/application.api-test.js

```diff
@@ -39,6 +40,11 @@ const mockSuccessfulResponse = {
   },
 };
 
+const expectedEligibilityCriteriaAuditRecord = {
+  ...generateParsedMockPortalUserAuditDatabaseRecord('abcdef123456abcdef123456'),
+  lastUpdatedByPortalUserId: expect.any(String),
```

Comment:
These ECs are uploaded by Mock Data Loader so we dont have access to the portal user id.